curator.py

```diff
@@ -10,7 +10,7 @@
 # read environment variables
 elasticsearch_host = os.getenv("ELASTICSEARCH_HOST", "elasticsearch:9200")
 retention_days = int(os.getenv("RETENTION_DAYS", "14"))
-index_name_prefix = os.getenv("INDEX_NAME_PREFIX", "fluentd-")
+index_name_prefix = os.getenv("INDEX_NAME_PREFIX", "fluentd- gslogs-")
```

Comment:
Unrelated to the bug fix: Setting the default to a sensible value (former prefix and future prefix).